### ISSUE TITLE
No game audio running Vita3k

### ISSUE THREAD

**Issue by t1murl** [dragoonDorise/EmuDeck#472]
Hi guys, thanks for the awesome work with Emudeck!

I was eager to try out vita3k now that it is added to emudeck with experimental support.
However, I find my games not playing any audio at all.

I followed the vita3k instructions on startup regarding firmware setup, installed my NoNpDRM games and ran Steam Rom Manager to be able to start games from Game Mode.

Games like Persona 4 Golden or Trails of Cold Steel startup just fine and are actually _somewhat_ playable (minor to major graphical glitches), however no ingame audio whatsoever.

Is this just me and is there anything I can do?

I'll gladly provide e.g. logs if given instructions in case this would be of any help.

Thanks!

**Comment by t1murl** [comment]
> I'd start with launching vita3k in desktop mode and ensuring that steamos has not muted the process, or set it to use echo-cancel-sink

I'll gladly try that and report back. Thanks for your very fast response @Godsbane :)

**Comment by Godsbane** [comment]
no prob - make sure you have vita3k playing content - or it won't show up in the volume control tray, i don't believe.

**Comment by t1murl** [comment]
I started vita3k in desktop mode and ran a game, then looked at the application audio menu in the task bar and noticed it did not look like audio was playing as the audio level bar did not move.
I tried lowering and raising the maximum volume for the application and the instant I touched it the in game audio started playing.

I rebooted into gaming mode and now there's sound in all games in vita3k!🙂 Also after a full shutdown and boot still working.

Whatever it was, apparently that fixed it.

Thanks for guiding me into the right direction @Godsbane !
